app/dashboard/templates/profiles/profile.html

```diff
@@ -204,6 +204,12 @@ <h1 class="profile-header__handle">
         </div>
       {% endif%}
 
+      <div class="container px-md-0">
+        <div class="row">
```

Comment:
only show this IFF it's the users own profile

Reply:
🤔